source/driver-bulk-update.rst

```diff
@@ -895,13 +899,7 @@ Test Case 2:
         
         Same for initializeOrderedBulkOp().    
 
-Test Case 3:
-    Key validation, no $-prefixed keys allowed:
-        batch = initializeUnorderedBulkOp().insert({$key: 1})
-        
-        bulk.execute() throws reasonable error (server does the validation)
-        
-        Same for initializeOrderedBulkOp().
+Test Case 3: *Removed*
```

Comment:
Why not just remove the test case entirely?

Reply:
This follows what was done in [Change Streams: Prose Tests](https://github.com/mongodb/specifications/tree/master/source/change-streams/tests#prose-tests). See https://github.com/mongodb/specifications/commit/66c6e9177709230483e914ec7773b5a996c89554 for an explanation.